diff --git a/lib/support/nginx/gitlab b/lib/support/nginx/gitlab
index 74eb8634d..8b440237e 100644
--- ./lib/support/nginx/gitlab
+++ ./lib/support/nginx/gitlab
@@ -19,7 +19,7 @@
 upstream gitlab-workhorse {
   # GitLab socket file,
   # for Omnibus this would be: unix:/var/opt/gitlab/gitlab-workhorse/sockets/socket
-  server unix:/home/git/gitlab/tmp/sockets/gitlab-workhorse.socket fail_timeout=0;
+  server localhost:8181 fail_timeout=0;
 }
 
 map $http_upgrade $connection_upgrade_gitlab {
@@ -68,21 +68,23 @@ server {
   ## the ip address of the server (http://x.x.x.x/)n 0.0.0.0:80 default_server;
   listen 0.0.0.0:80 default_server;
   listen [::]:80 default_server;
-  server_name YOUR_SERVER_FQDN; ## Replace this with something like gitlab.example.com
   server_tokens off; ## Don't show the nginx version number, a security best practice
 
   ## See app/controllers/application_controller.rb for headers set
 
   ## Real IP Module Config
   ## http://nginx.org/en/docs/http/ngx_http_realip_module.html
-  real_ip_header X-Real-IP; ## X-Real-IP or X-Forwarded-For or proxy_protocol
-  real_ip_recursive off;    ## If you enable 'on'
+  real_ip_header X-Forwarded-For; ## X-Real-IP or X-Forwarded-For or proxy_protocol
+  real_ip_recursive on;    ## If you enable 'on'
   ## If you have a trusted IP address, uncomment it and set it
   # set_real_ip_from YOUR_TRUSTED_ADDRESS; ## Replace this with something like 192.168.1.0/24
+  set_real_ip_from  10.0.0.0/8;
+  set_real_ip_from  172.16.0.0/12;
+  set_real_ip_from  192.168.0.0/16;
 
   ## Individual nginx logs for this GitLab vhost
-  access_log  /var/log/nginx/gitlab_access.log gitlab_access;
-  error_log   /var/log/nginx/gitlab_error.log;
+  access_log  /dev/stdout gitlab_access;
+  error_log   /dev/stdout;
 
   location / {
     client_max_body_size 0;
@@ -98,6 +100,7 @@ server {
 
     proxy_set_header    Host                $http_host;
     proxy_set_header    X-Real-IP           $remote_addr;
+    proxy_set_header    X-Forwarded-Ssl     on;
     proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
     proxy_set_header    X-Forwarded-Proto   $scheme;
     proxy_set_header    Upgrade             $http_upgrade;
