--- ./config/puma.rb.example
+++ ./config/puma.rb.example
@@ -8,9 +8,9 @@ rackup 'config.ru'
 pidfile '/home/git/gitlab/tmp/pids/puma.pid'
 state_path '/home/git/gitlab/tmp/pids/puma.state'
 
-stdout_redirect '/home/git/gitlab/log/puma.stdout.log',
-  '/home/git/gitlab/log/puma.stderr.log',
-  true
+# stdout_redirect '/home/git/gitlab/log/puma.stdout.log',
+#   '/home/git/gitlab/log/puma.stderr.log',
+#   true
 
 # Configure "min" to be the minimum number of threads to use to answer
 # requests and "max" the maximum.
