diff --git a/config/resque.yml.example b/config/resque.yml.example
index 600b9c67d6fa..2a473782f99b 100644
--- a/config/resque.yml.example
+++ b/config/resque.yml.example
@@ -1,8 +1,8 @@
 # If you change this file in a merge request, please also create
 # a merge request on https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests
 #
-development:
-  url: redis://localhost:6379
+#development:
+#  url: redis://localhost:6379
   # ssl_params:
   #   ca_path: "/path/to/dir/with/certs"
   #   ca_file: "/path/to/ca.crt"
@@ -15,8 +15,8 @@ development:
   #   -
   #     host: replica2
   #     port: 26381 # point to sentinel, not to redis port
-test:
-  url: redis://localhost:6379
+#test:
+#  url: redis://localhost:6379
 production:
   # Redis (single instance)
   url: unix:/var/run/redis/redis.sock
