--- ./config/resque.yml.example
+++ ./config/resque.yml.example
@@ -1,8 +1,8 @@
 # If you change this file in a Merge Request, please also create
 # a Merge Request on https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests
 #
-development:
-  url: redis://localhost:6379
+#development:
+#  url: redis://localhost:6379
   # sentinels:
   #   -
   #     host: localhost
@@ -10,8 +10,8 @@ development:
   #   -
   #     host: replica2
   #     port: 26381 # point to sentinel, not to redis port
-test:
-  url: redis://localhost:6379
+#test:
+#  url: redis://localhost:6379
 production:
   # Redis (single instance)
   url: unix:/var/run/redis/redis.sock
