diff --git a/config.toml.example b/config.toml.example
index d878b16ad..281345eec 100644
--- a/config.toml.example
+++ b/config.toml.example
@@ -2,10 +2,10 @@
 # For Gitaly documentation, see https://docs.gitlab.com/ee/administration/gitaly/
 
 # A path which Gitaly should open a Unix socket.
-socket_path = "/home/git/gitlab/tmp/sockets/private/gitaly.socket"
+socket_path = "/home/git/run/gitaly/gitaly.socket"
 
 # Directory containing Gitaly executables.
-bin_dir = "/home/git/gitaly/_build/bin"
+bin_dir = "/usr/local/bin"
 
 # # Optional. The directory where Gitaly can create all files required to
 # # properly operate at runtime. If not set, Gitaly will create a directory in
@@ -83,8 +83,8 @@ dir = "/home/git/gitlab/log"
 custom_hooks_dir = "/home/git/custom_hooks"
 
 [gitlab]
-secret_file = "/home/git/gitlab-shell/.gitlab_shell_secret"
-url = "http+unix://%2Fhome%2Fgit%2Fgitlab%2Ftmp%2Fsockets%2Fgitlab-workhorse.socket"
+secret_file = "/etc/gitlab/gitlab-shell/secret/gitlab_shell_secret"
+url = "http+unix://%2Fhome%2Fgit%2Frun%2Fgitlab%2Fgitlab.socket"
 # Only needed if a UNIX socket is used in `url` and GitLab is configured to
 # use a relative path (e.g. /gitlab).
 # relative_url_root = '/'
