diff --git a/config.toml.example b/config.toml.example
index 82b8502f8..97bea37d5 100644
--- a/config.toml.example
+++ b/config.toml.example
@@ -2,10 +2,10 @@
 # For Gitaly documentation, see https://docs.gitlab.com/ee/administration/gitaly/.
 
 # A path which Gitaly should open a Unix socket.
-socket_path = "/home/git/gitlab/tmp/sockets/private/gitaly.socket"
+socket_path = "/home/git/run/gitaly/gitaly.socket"
 
 # Directory containing Gitaly executables.
-bin_dir = "/home/git/gitaly/_build/bin"
+bin_dir = "/usr/local/bin"
 
 # # Optional. The directory where Gitaly can create all files required to
 # # properly operate at runtime. If not set, Gitaly will create a directory in
@@ -96,13 +96,13 @@ custom_hooks_dir = "/home/git/custom_hooks"
 # # Gitaly must connect to the GitLab application to perform access checks when a user performs a change.
 [gitlab]
 # # URL of the GitLab server.
-url = "http+unix://%2Fhome%2Fgit%2Fgitlab%2Ftmp%2Fsockets%2Fgitlab-workhorse.socket"
+url = "http+unix://%2Fhome%2Fgit%2Frun%2Fgitlab%2Fgitlab.socket"
 # # 'relative_url_root' is only needed if a UNIX socket is used in 'url' and GitLab is configured to
 # # use a relative path. For example, '/gitlab'.
 # relative_url_root = '/'
 # # Path of the file containing the secret token used to authenticate with GitLab. Use either 'secret_token' or 'secret'
 # # but not both.
-secret_file = "/home/git/gitlab-shell/.gitlab_shell_secret"
+secret_file = "/etc/gitlab/gitlab-shell/secret/gitlab_shell_secret"
 # # Secret token used to authenticate with GitLab.
 # secret = ""
 
