diff --git a/lib/support/logrotate/gitlab b/lib/support/logrotate/gitlab
index 43fa8eb963c7..113e81ab472a 100644
--- a/lib/support/logrotate/gitlab
+++ b/lib/support/logrotate/gitlab
@@ -4,18 +4,9 @@
     su git git
     daily
     missingok
-    rotate 90
-    compress
-    notifempty
-    copytruncate
-}
-
-/home/git/gitlab-shell/gitlab-shell.log {
-    su git git
-    daily
-    missingok
-    rotate 90
+    rotate 14
     compress
     notifempty
     copytruncate
+    maxsize 100M
 }
