--- ./config/unicorn.rb.example
+++ ./config/unicorn.rb.example
@@ -64,8 +64,8 @@ pid "/home/git/gitlab/tmp/pids/unicorn.pid"
 # By default, the Unicorn logger will write to stderr.
 # Additionally, some applications/frameworks log to stderr or stdout,
 # so prevent them from going to /dev/null when daemonized here:
-stderr_path "/home/git/gitlab/log/unicorn.stderr.log"
-stdout_path "/home/git/gitlab/log/unicorn.stdout.log"
+# stderr_path "/home/git/gitlab/log/unicorn.stderr.log"
+# stdout_path "/home/git/gitlab/log/unicorn.stdout.log"
 
 # Save memory by sharing the application code among multiple Unicorn workers
 # with "preload_app true". See:
